### PR TITLE
PlottingGL: picking fix

### PR DESCRIPTION
**Pull request by martinfalk**
* fixes handling of picking events which are not managed by the respective plots

### CODE REVIEW

File: modules/plottinggl/src/plotters/scatterplotgl.cpp [hunk 1]
```diff
@@ -492,24 +492,26 @@ void ScatterPlotGL::objectPicked(PickingEvent *p) {
...
-
-            if (touchPoint.state() == TouchState::Started) {
-                // print information on current element
-                logRowData();
+        } else if (auto touchEvent = p->getEventAs<TouchEvent>()) {
+            if (touchEvent->touchPoints().size() == 1) {
+                // allow interaction only for a single touch point
+                const auto &touchPoint = touchEvent->touchPoints().front();
+
+                if (touchPoint.state() == TouchState::Started) {
```
Comment (ResearchDaniel) [inline modules/plottinggl/src/plotters/scatterplotgl.cpp]: Not part of this PR but it should be TouchState:: Finished?

File: modules/plottinggl/src/plotters/scatterplotgl.cpp [hunk 2]
```diff
@@ -492,24 +492,26 @@ void ScatterPlotGL::objectPicked(PickingEvent *p) {
         }
     };
 
-    if (auto me = p->getEventAs<MouseEvent>()) {
-        if (me->button() == MouseButton::Left) {
-            if (me->state() == MouseState::Release) {
-                // print information on current element
-                logRowData();
+    if (p->getState() == PickingState::Updated) {
+        if (auto me = p->getEventAs<MouseEvent>()) {
```
Comment (ResearchDaniel) [inline modules/plottinggl/src/plotters/scatterplotgl.cpp]: Separate handling of mouse and touch can be remove if we use this?
```
p->getState() == PickingState::Updated && p->getPressState() == PickingPressState::Release &&
        p->getPressItem() == PickingPressItem::Primary
```

**Review by ResearchDaniel — APPROVED** [review]
Looks fine to me. Some related comments though

File: modules/plottinggl/src/plotters/scatterplotgl.cpp (hunk 2 above)
Comment (martinfalk) [reply modules/plottinggl/src/plotters/scatterplotgl.cpp]: might be possible

**Review by petersteneteg — APPROVED** [review]
